Controllers/DeleteTemplate.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Leantime\Plugins\TicketTemplate\Controllers;
+
+use Leantime\Core\Controller;
+use Leantime\Core\Frontcontroller;
+use Leantime\Domain\Auth\Models\Roles;
+use Leantime\Domain\Auth\Services\Auth;
+use Leantime\Plugins\TicketTemplate\Repository\TicketTemplateRepository;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Create template Controller for TicketTemplate plugin
+ *
+ * @package    leantime
+ * @subpackage plugins
+ */
+class DeleteTemplate extends Controller
+{
+    /**
+     * run - delete template.
+     *
+     * @return Response
+     */
+    public function run(): Response
+    {
+        Auth::authOrRedirect([Roles::$owner, Roles::$admin], true);
+
+        $ticketTemplateRepository = app()->make(TicketTemplateRepository::class);
+
+        if (isset($_GET['id']) === true) {
```

Comment:
Why this `=== true` isset retrun true or a value that will be false in an if statement?